A32NX/html_ui/Pages/A32NX_Core/A32NX_FADEC.js

```diff
@@ -0,0 +1,42 @@
+class A32NX_FADEC {
+    constructor(_engine) {
+        this.engine = _engine;
+        this.fadecTimer = -1;
+    }
+
+    init() {
+        this.updateSimVars();
+    }
+
+    update(_deltaTime) {
+        const dcAvail = SimVar.GetSimVarValue("L:DCPowerAvailable", "Bool");
+        const masterState = SimVar.GetSimVarValue("FUELSYSTEM VALVE SWITCH:"+(this.engine+5), "Bool");
+        const ignitionState = SimVar.GetSimVarValue("L:XMLVAR_ENG_MODE_SEL", "Enum") == 2;
+        if (this.lastDCState != dcAvail && dcAvail == 1) {
+            this.fadecTimer = 5*60;
+        }
+        if (this.lastMasterState != masterState) {
```

Comment:
Could condense this by replacing line 15 with:
```
 if ((this.lastDCState != dcAvail && dcAvail == 1) || this.lastMasterState != masterState) {
```

Reply:
Implemented.